src/plugins/profiling/common/index.ts

```diff
@@ -85,3 +87,9 @@ export function groupSamplesByCategory(samples) {
   }
   return series;
 }
+
+export function timeRangeFromRequest(request: any): [number, number] {
```

Comment:
I just wonder if we can make use of the function `parseInterval` in `src/plugins/data/common/search/aggs/utils/date_interval_utils/parse_interval.ts`.

Reply:
mhmh not sure that one returns a `Duration`, while we will use `number` in the queries to Elasticsearch

Reply:
Otherwise it looks good to me.

Reply:
ah I see now the `Duration.seconds()` method, let me try that